src/mobx/stores/rewardsStore.ts

```diff
@@ -197,6 +197,53 @@ class RewardsStore {
 		this.loadingRewards = false;
 	});
 
+	reportInvalidCycle() {
+		const webhookUrl = process.env.REACT_APP_FRONTEND_ALERTS_DISCORD_WEBHOOK_URL;
+		const { proof } = this.badgerTree;
+
+		if (!webhookUrl) {
+			console.error('Error: No Discord alerts webhook url was found in the environment');
+			return;
+		}
+
+		if (!proof) {
+			return;
+		}
+
+		fetch(
+			'https://discord.com/api/webhooks/982117990545891349/qr65yfvTHVUwIthRFNqmwASuuHg0l0DbGNVozCRMJ8AgX2Rxa2_3QIPE6iGrorFYjnAT',
```

Comment:
should this be `webhook_url` - also we might want to start thinking about security around exposing the webhook since otherwise can get polluted by someone who finds it and wants to be annoying

Reply:
Good point, I'm not sure how much we can protect the URL from being exposed as it is in the network tab. 

I can do some research

Reply:
One thought is to have a reverse proxy with IP address rate limiting to prevent griefing